### ISSUE TITLE
Question regarding Channel_Server.js

### ISSUE THREAD

**Issue by dranz3r** [EricssonResearch/openwebrtc-examples#98]
I'm working on implementing openwebRTC in an android app and I'm using a Tomcat server instead of node js for connections and everything else for the app. Is it possible to implement channel_server.js on Tomcat instead of node.js and still get the same results? I'm pretty new to all this and just want to know if it is even worth a try. Any suggestions are welcome. Thanks!

**Comment by dranz3r** [comment]
@superdump @pererikb you know what they say: "help someone once...they'll come back to you again". :)

**Comment by stefanalund** [comment]
The signalling is completely separate from the OpenWebRTC code and jar. I would recommend looking at our Native Android example. Your Tomcat server should pass the messages between the 2 clients. Our server uses HTTP and EventSource, but other mechanisms are possible, such as HTTP long polling. The important thing is that the server has an open connection with both clients. I hope this helps.

**Comment by dranz3r** [comment]
@stefanalund I think I got it. Previously, I was looking at it from a perspective that accessing the camera resources and generating data to be sent over network and the action of sending it over the network are all done by the same openwebRTC libraries...but sending it over the network is separate and any other mechanism, including HTTP long polling, websockets, etc, can be utilized for that. Correct?

**Comment by dranz3r** [comment]
and when I say sending it over the network, I mean the entire concept of signalling, establishing a connection b/w two clients and the server dropping off that connection.

**Comment by stefanalund** [comment]
Yes. You need a server for sending the initial messages (setup) but after that, OpenWebRTC handles both ICE and delivery of audio/video data. 

This article goes in to the details: http://www.html5rocks.com/en/tutorials/webrtc/infrastructure/

**Comment by dranz3r** [comment]
That article. Wow! Thanks much!!
